.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "manifests/opmon/grafana/grafana-dashboards"]
+	path = manifests/opmon/grafana/grafana-dashboards
+	url = https://github.com/DUNE-DAQ/grafana-dashboards.git
```

Comment:
The path to the graphana repository should be relative to use the same authentication as pocket when cloned.

Reply:
I can change the url to SSH clone instead. I don't know how to change it relative to the way pocket is cloned, but will investigate

Reply:
`url = ../grafana-dashboards.git` should work